### DISCUSSION TITLE
going from base class instance to derived class instance?

### DISCUSSION THREAD

**Discussion by pcompassion** [jazzband/django-polymorphic#702] [Q&A] [answered]
The documentation illustrates how you can work with model's manager class to get derived class results.

How about individual model instance?  
Does django_polymorphic offer a way to get to a derived model instance from a base model instance?

**Comment by vdboor** [comment, marked as answer]
Yes, there is a `get_real_instance()` method available.

Clearly, this needs to be emphesized in the docs. If you know a good place for it, please provide us a pull request!
